config/samples/gatling-operator_v1alpha1_gatling01.yaml

```diff
@@ -3,10 +3,12 @@ kind: Gatling
 metadata:
   name: gatling-sample01
 spec:
-  generateReport: false                                   # The flag of generating gatling report
+  generateReport: true                                   # The flag of generating gatling report
+  #generateReport: false                                   # The flag of generating gatling report
```

Comment:
In this sample, I think `generateReport` is intended to be false.

Reply:
you're light!  it's just for testing